### DISCUSSION TITLE
Enviroment SERVICE_FQDN_SERVICENAME forward slash Problem

### DISCUSSION THREAD

**Discussion by Jopo-JP** [coollabsio/coolify#2572] [Improvement Requests]
I would like have an option or an env to remove the trailing slash at the end.

Example:
$SERVICE_FQDN_PAPERLESS

Result:
https://paperless.domain.com/

The required output is:
https://paperless.domain.com

Some services don't like it and will refuse to work. I wanted to create a template for Paperless-NGX for Coolify templates, everthing is working, but the user needs to input the full url manually, as django won't remove the / at the end.

Is there maybe a workaround?

**Comment by martonsz** [comment]
I created a PR with a workaround: https://github.com/coollabsio/coolify/pull/2719

**Comment by Jopo-JP** [comment]
Thanks

**Comment by peaklabs-dev** [comment]
Just for information, the workaround is no longer needed as we have introduced a new parser that fixes this problem.
